torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -317,6 +379,8 @@ def decompress(fut):
             state.p_memory_dict.clear()
             state.q_memory_dict.clear()
 
+        state.maybe_increase_iter(bucket)
```

Comment:
Nit: do we need to continue incrementing iter here? Currently once we start powerSGD we never go back, so we can omit this?

Reply:
Agree it's not that important. Also, I don't think there is any harm to keep incrementing the counter here.

Don't want to overcomplicate the counter increment logic here. I would prefer keeping incrementing here, which can be useful for debugging and potential future development, such as mixing more than 2 compression schemes.